app/config/security.yml

```diff
@@ -39,6 +39,7 @@ security:
             logout:
                 path: security_logout
                 target: pelagos_homepage
+                handlers: [pelagos.logout_listener]
```

Comment:
It's not really clear from documentation, but it looks like `handlers` will go way in Symfony 4.
You probably wanna use `success_handler` anyway because that doesn't just handle the logout, and actual logout success. And in that case you want to implement a `LogoutSuccessHandlerInterface`

Reply:
Also, it's not a listner, but a handler.

Reply:
Turns out success_handler is called before logout handler, so no need to change it.